google/cloud/spanner_v1/pool.py

```diff
@@ -151,8 +162,14 @@ class FixedSizePool(AbstractSessionPool):
     DEFAULT_SIZE = 10
     DEFAULT_TIMEOUT = 10
 
-    def __init__(self, size=DEFAULT_SIZE, default_timeout=DEFAULT_TIMEOUT, labels=None):
-        super(FixedSizePool, self).__init__(labels=labels)
+    def __init__(
+        self,
+        size=DEFAULT_SIZE,
+        default_timeout=DEFAULT_TIMEOUT,
+        labels=None,
+        creator_role=None,
+    ):
+        super(FixedSizePool, self).__init__(labels=labels, creator_role=creator_role)
```

Comment:
Somewhere over here, if the value is None, should we not check the database for the default value?
Same for other types of pools?